runtimes/rust/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "exonum-rust-runtime"
+version = "0.13.0-rc.2"
+edition = "2018"
+authors = ["The Exonum Team <exonum@bitfury.com>"]
+homepage = "https://exonum.com/"
+repository = "https://github.com/exonum/exonum"
+documentation = "https://docs.rs/exonum-rust-runtime"
+readme = "README.md"
+license = "Apache-2.0"
+keywords = ["exonum", "runtime"]
+categories = ["cryptography"]
+description = "The runtime is for running native services written in Rust."
+
+[badges]
+travis-ci = { repository = "exonum/exonum" }
+
+[dependencies]
+exonum = { version = "0.13.0-rc.2", path = "../../exonum" }
+exonum-crypto = { version = "0.13.0-rc.2", path = "../../components/crypto" }
+exonum-derive = { version = "0.13.0-rc.2", path = "../../components/derive" }
```

Comment:
Since derive macros are part of a public interface for a rust runtime, shouldn't we provide public re-exports for required macros?

Reply:
How I can do this? If I add `pub use exonum_derive::*` I get error.

```
the name `ServiceFactory` is defined multiple times
`ServiceFactory` must be defined only once in the macro namespace of this module
```